azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+jobs:
+
+- job: 'Test'
+  pool:
+    vmImage: 'Ubuntu 16.04'
```

Comment:
I think we may need to check code on other platforms (at least macOS) as well

Reply:
Added macOS to the pool, it seems to be working